package.json

```diff
@@ -16,5 +16,9 @@
   "scripts": {
     "dev": "gulp dev",
     "prestart": "gulp"
+  },
+  "engines": {
+    "node": ">=8.9.1",
```

Comment:
There was not an engine information for the node version, so I set it to the version that we use. I can replace it whatever version that this project normally assumes.